tests/ui/pages/desktop/details.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from base import Base
+
+
+class Details(Base):
+    """ Details page. """
+
+    _wait_element = (By.CLASS_NAME, 'addon-details')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_all_elements_located(
+            (self._wait_element)))
+        return self
+
+    @property
+    def header(self):
+        return self.Header(self)
+
+    class Header(Region):
```

Comment:
I don't know that we need a header region, and I believe this is overwriting the header property from the base page object. Perhaps an DescriptionHeader region could make sense, or just skip the region entirely and add properties directly to the page object.